alumet/src/plugin/event.rs

```diff
@@ -155,8 +163,13 @@ pub struct StartConsumerMeasurement(pub Vec<ResourceConsumer>);
 #[derive(Clone)]
 pub struct StartResourceMeasurement(pub Vec<Resource>);
 
+/// Event occurring when measurements should be performed at the end of the consumer experiment.
+#[derive(Clone)]
+pub struct EndConsumerMeasurement();
```

Comment:
Since this struct carries no data, you can declare it like this:
```rs
pub struct EndConsumerMeasurement;
```
and use it like this: `EndConsumerMeasurement` (without the parentheses).